package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Comment:
I think you added a dependency when you were at the root level of the project. You need to enter the webapp directory and install it

Reply:
Is that the correct way? @JustinCast